projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -73,6 +75,23 @@ public function add_browse_sites_link() {
 		add_filter( 'add_menu_classes', array( $this, 'set_browse_sites_link_class' ) );
 	}
 
+	/**
+	 * Test_Admin_Menu.
+	 *
+	 * @covers ::reregister_menu_items
+	 */
+	public function test_admin_menu_output() {
+		global $menu;
+
+		static::$admin_menu->reregister_menu_items();
+
+		$this->assertSame(
+			array_keys( $menu ),
+			array( 2, '3.86682', 4, 5, 10, 15, 20, 25, 50, 51, 59, 60, 61, 65, 70, 75, 80 ),
+			'Admin menu should not have unexpected top menu items.'
+		);
+	}
```

Comment:
Should this be in `test-class-wpcom-admin-menu.php`?

Reply:
Ahaha thanks, I was having some copy pasta issues. Gonna just remove it altogether.